projects/scannet_offline_eval/configs/model/instancemap3d_top_down.yaml

```diff
@@ -3,6 +3,6 @@ defaults:
   - semantic_sensor: grounded_sam_perception
   - voxel_map: default
 
-_target_: build_sparse_voxel_map.SparseVoxelMapAgent
+_target_: home_robot.agent.multitask.sparse_voxel_instance_map.SparseVoxelMapAgent
```

Comment:
what is this file doing?

Reply:
This implements `build_representation_and_return_queries` -- it's the "final form" that consumes posed RGBD frames and text, and returns a response. The style that the rest of the benchmark tasks will follow